style.css

```diff
@@ -0,0 +1,72 @@
+* {
+  font-family: 'Poppins', sans-serif;
+}
+
+nav {
+  display: flex;
+  margin: 20px;
+}
+
+.hamburger {
+  float: right;
+  margin-left: 60%;
+}
+
+body {
+  background-image: url("assets/images/shapesbg.svg");
+  background-repeat: no-repeat;
+}
```

Comment:
- [ ] [OPTIONAL] This will not be the only time you'll use a background image. Is is better to use background image styling on the single section and not the whole body, otherwise the image will be stretched as the project grows in size. Right now, this image is only required for the top section.